clients/cpp/README.md

```diff
@@ -0,0 +1,6 @@
+ASP.NET Core SignalR C++ Client
+========
+
+This folder contains a C++ client for ASP.NET Core SignalR.
+
+**This is an experimental project right now**
```

Comment:
Let's make it even clearer and add "There are no plans to ship this client at this time"